.github/workflows/CI.yml

```
@@ -21,6 +21,7 @@ jobs:
     strategy:
       matrix:
         version:
+          - '1.7'
```

A bit more future-proof:
```suggestion
          - 'min'
```

---

Ah, nice, quite new that feature.